### DISCUSSION TITLE
Changing confirmation email template's title not working

### DISCUSSION THREAD

**Discussion by hallee9000** [supabase/supabase#2320] [Questions] [answered]
I changed the confirmation email template's title.
![image](https://user-images.githubusercontent.com/12868055/125197631-0fe91180-e291-11eb-8ace-3ac2dbc591e1.png)

But it's still the old one in my inbox.
![image](https://user-images.githubusercontent.com/12868055/125197616-f8118d80-e290-11eb-94b9-6efe46176df1.png)

BTW can we change the email sender name? `noreply` looks like meaningless. Maybe the project's name or just `supabase` is fine.
![image](https://user-images.githubusercontent.com/12868055/125197723-71a97b80-e291-11eb-927f-5fc4f680d810.png)

**Comment by ChronSyn** [comment]
I'm having an issue that might be related - attempting to change the redirect and site URL's via the dashboard is throwing an error 500. I've sent an email over to the team to see if they're aware and if they're able to implement a fix.

**Comment by mryechkin** [comment]
Experiencing this issue as well, but with the Reset Password template. In my case I'm updating the `body` of the template but it's not taking effect

**Reply by kangmingtay** [reply]
Hi @mryechkin , hmm i'm unable to reproduce the issue you're facing and i'm able to update the header and body.

![image](https://user-images.githubusercontent.com/28647601/127260742-ca06d119-81a2-4bd3-91c7-71d75ad950fe.png)
![image](https://user-images.githubusercontent.com/28647601/127260766-db88959a-05c4-484b-976c-7a7adc78e116.png)

**Reply by kangmingtay** [reply]
Could you kindly double check if there are any syntax errors in the template?

**Reply by mryechkin** [reply]
@kangmingtay it appears it's because I was trying to use a `<table>` with some custom inline styling within the template body? Seems to work otherwise. What all is allowed to go in the template body?

**Reply by mryechkin** [reply]
OK, it seems to be working now after changing it up some more. Must've been a syntax error somewhere, as you said. Thanks! Glad to see it working. Onto the next step :)

**Comment by kangmingtay** [comment, marked as answer]
Hey @leadream, I just tried to reproduce your issue with change the confirmation email template title and it seems to work fine for me. Let me know if you're still facing this issue. You can set your custom smtp config and change the email sender name if you go to Settings -> Auth Settings -> ENABLE CUSTOM SMTP -> SMTP Sender Name

**Reply by hallee9000** [reply]
Thanks @kangmingtay , I'll try it later.

**Reply by hallee9000** [reply]
Great! It works. Thanks @kangmingtay
